zeus/metric.py

```diff
@@ -54,6 +73,9 @@ class EnergyHistogram(Metric):
         gpu_bucket_range: Histogram buckets for GPU energy.
         cpu_bucket_range: Histogram buckets for CPU energy.
         dram_bucket_range: Histogram buckets for DRAM energy.
+        gpu_histograms: A single Prometheus Histogram metric for all GPU energy consumption, indexed by window and GPU index.
+        cpu_histograms: A single Prometheus Histogram metric for all CPU energy consumption, indexed by window and CPU index.
+        dram_histograms: A single Prometheus Histogram metric for all DRAM energy consumption, indexed by window and DRAM index.
```

Comment:
Remove the entire `Attributes` section. They're not intended to be public attributes AFAIK.